### PR TITLE
[app] Initialise WebComponents and Lit

### PR DESCRIPTION
**Pull request by zerefwayne**
This PR initialises WebComponents and adds Lit as a dependency to the project.

To be merged after #1

### CODE REVIEW

File: diagnostics-app/src/index.html [hunk 1]
```diff
@@ -4,22 +4,37 @@
...
-</body>
+  <head>
+    <meta charset="utf-8" />
+    <title>DiagnosticsApp</title>
+    <base href="/" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" type="image/x-icon" href="favicon.ico" />
+    <script src="webcomponents/webcomponents-loader.js"></script>
+    <script>
+      if (!window.customElements) {
```
Comment (Lamzin) [inline diagnostics-app/src/index.html]: Why do we need it?

File: diagnostics-app/src/app/app.component.css [hunk 2]
```diff
@@ -3,8 +3,3 @@
  * Use of this source code is governed by a BSD-style license that can be
  * found in the LICENSE file.
  */
-
```
Comment (Lamzin) [inline diagnostics-app/src/app/app.component.css:6]: Is this cleanup related to the PR?

File: diagnostics-app/src/app/app.component.html [hunk 3]
```diff
@@ -4,9 +4,4 @@
  found in the LICENSE file.
 -->
 
-<!--
```
Comment (Lamzin) [inline diagnostics-app/src/app/app.component.html:7]: same

**Review by Lamzin — APPROVED** [review]
LGTM % comments

File: diagnostics-app/src/app/app.component.css (hunk 2 above)
Comment (zerefwayne) [reply diagnostics-app/src/app/app.component.css:6]: Yes, this is for cleanup!

File: diagnostics-app/src/app/app.component.html (hunk 3 above)
Comment (zerefwayne) [reply diagnostics-app/src/app/app.component.html:7]: Yes, cleanup.

File: diagnostics-app/src/index.html (hunk 1 above)
Comment (zerefwayne) [reply diagnostics-app/src/index.html]: @Lamzin Yes, this was not needed. As Webcomponents will be available in the browser that we intend to target. Removed this script tag.

**Review by MahmoudAGawad — APPROVED** [review]
LGTM. Thanks!

**Comment by zerefwayne** [comment]
Merging the PR! :+1:
